repo/packages/W/weavescope/2/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "packagingVersion": "3.0",
+  "name": "weavescope",
+  "version": "1.2.1",
+  "tags": ["microservices", "visualisation", "monitoring"],
+  "maintainer": "help@weave.works",
+  "description": "Live visualisation, monitoring and management\n\nWeave Scope produces a live interactive map of your containerized services and applications with no coding and no instrumentation",
```

Comment:
@errordeveloper A documentation example is helpful for new and existing users to understand how they can install and run a package on DC/OS. This is also now part of package submission guidelines: https://github.com/mesosphere/universe#submit-your-package
It shouldn't take a lot of your time as you can use the template the existing examples. 
Ex: https://github.com/dcos/examples/tree/master/1.8/flink